.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Please add the test.md file to your .gitignore so that the file is not pushed to GitHub. 
add this line to your .gitignore `test.md`
![copy](https://user-images.githubusercontent.com/99656687/226590568-40917a0b-facb-4958-a8f1-27501c67d6f8.png)